app/controllers/invoices_controller.rb

```diff
@@ -32,7 +32,8 @@ def index
 
     return unless params[:state] == 'payment'
 
-    message = 'Payment was successfully created. Your payment will be processed as soon as possible. Thank you!'
+    message = 'Payment was successfully created. ' \
+      'Your payment will be processed as soon as possible. Thank you!'
```

Comment:
I forgot to correct one flaw here. It would be better if this message was localised.

Reply:
Done!